src/SmartComponents/Systems/SystemsTable.js

```diff
@@ -188,7 +181,7 @@ const SystemsTable = ({
                 actionResolver: (row) =>
                     systemsRowActions(
                         activateRemediationModal,
-                        !templateUpdateEnabled  && openAssignSystemsModal,
+                        false,
```

Comment:
nit-pick: we could do even more clean-up here by removing the second argument completely from the function and always return only one action "Apply all applicable advisories" from systemsRowActions

Reply:
Yes, this will come in a subsequent PR, I didn't want this one to be so bloated.